src/main/java/games/strategy/triplea/TripleA.java

```diff
@@ -65,11 +65,8 @@
         players.add(new ProAI(name, type));
       } else if (type.equals(DOESNOTHINGAI_COMPUTER_PLAYER_TYPE)) {
         players.add(new DoesNothingAI(name, type));
-      } else if (type.equals(HUMAN_PLAYER_TYPE) || type.equals(CLIENT_PLAYER_TYPE)) {
-        final TripleAPlayer player = new TripleAPlayer(name, type);
-        players.add(player);
       } else {
-        throw new IllegalStateException("Player type not recognized:" + type);
+        players.add(new TripleAPlayer(name, type));
```

Comment:
Is this safe in the event someone adds a new player type in the future?

Reply:
I think the problem was that I had added a new player type and wanted this behavior. I'll need to double check on the motiviation. It was about 2 months now since I made this update.